Britannia/localization/english/BTWK_storyline_events_l_english.yml

```diff
@@ -1,36 +1,72 @@
+ BTWK_story_event.0102.a:0 "I shall honour his request"
+ BTWK_story_event.0102.b:0 "He is no king of mine."
 
  BTWK_story_event.0103.t:0 "The High Council at Glevum"
- BTWK_story_event.0103.desc:0 "After many night on the road, you finally arrive at the council in Glevum - probably could add a dynamic list telling you which of the relevant kings is present"
- BTWK_story_event.0103.a:0 "BEGIN"
- BTWK_story_event.0103.b:0 "No Father"
-
- BTWK_story_event.0104.t:0 "The High Council at Glevum - Norwenna"
- BTWK_story_event.0104.desc:0 "The first matter to attend to is a new foster father"
- BTWK_story_event.0104.a:0 "Powys"
- BTWK_story_event.0104.b:0 "Siluria"
- BTWK_story_event.0104.c:0 "Kernow"
- BTWK_story_event.0104.d:0 "Gwent"
- BTWK_story_event.0104.e:0 "Glevum"
- BTWK_story_event.0104.f:0 "Belgae"
- BTWK_story_event.0104.g:0 "Cynwidion"
- BTWK_story_event.0104.h:0 "Demetia"
- BTWK_story_event.0104.i:0 "Arthur"
-
- BTWK_story_event.0105.t:0 "The High Council at Glevum - Guardians"
- BTWK_story_event.0105.desc:0 "New desc"
+ BTWK_story_event.0103.desc:0 "After many a night on the road, you finally arrive at Glevum. DESCRIPTION OF GLEVUM.\n\n With the warlords now seated within the chamber, the council may now commence. "
+ BTWK_story_event.0103.a:0 "Let us find Mordred an ideal fosterer."
+ BTWK_story_event.0103.b:0 "Let us find Mordred ideal guardians."
+
+ BTWK_story_event.0118.t:0 "The Question of Arthur"
+ BTWK_story_event.0118.desc:0 "A skirmish in the Harvest season of 479 between Dumnonian forces and the Saxons concluded in the death of King Uther's only legitimate son and heir, Prince Mordred. However, upon hearing the news, Uther banished Prince Arthur - a bastard child of his - to King Ban's realm in Armorica, blaming the young soldier for his brother's untimely death.\n\n While this arrangement may have proven a boon for the Armorican ruler, the impending council could be the perfect opportunity to petition for Arthur's return. You will leave this task to your trusted Marshall, Comes Agricola, to represent the will of the martial class. Directly announcing your support for the Prince personally will not lend you any favours, least of all with Uther himself.\n\n The High King's hositilities aside, no one can deny Arthur is an exceptional man, soldier, and diplomat. Britannia may have capable men ready to fight for their country to the bitter end, history has taught you such people are doomed to oblivion without a skilled leader to match. Maybe Arthur is the answer?"
```

Comment:
- Harvest season -> fall/autumn? Unless they actually use "harvest season" in the books
- "banished Prince Arthur - a bastard child of his - to King Ban's realm in Armorica" -> "banished his bastard son Arthur to Benoic", I'm not sure if he's actually recognized as a prince in the books? Though you probably know better
- Marshall -> marshal
- May be better to localize "Comes Agricola" dynamically by getting his name/title/etc. directly from the character
- "to represent the will of the martial class" - not sure what this is referring to tbh?

Reply:
1. Don't think so, don't know why I used harvest season tbf it sounds a bit crap. Will change it to Autumn.
2. I did read ahead while on holiday, I'm not sure if he is or not. Pg. 229 has Gereint refer to him specifically as "Prince" in reference to him returning to Benoic to fulfil his oath, but Arthur himself typically hates being called Prince. On the flip side, the character sheet at the start of the book only refers to him as "Bastard son of Uther and protector of Mordred", so it could go either way. Based on the fact that he would not have the same legitimacy as an actual Prince heir, I reckon Gereint said this as a sign of respect, rather than anything that had legal bearing (or just a mistake by Bernard). Otherwise, there is also the issue of him being named "Pendragon" in the return event (a remnant copied from CK2). This doesn't happen in the immediate aftermaths of Uther's death (though I believe he was nicknamed Pendragon either later in TWK or later in the book series). I had a thought that, for the duration of him _not_ being named "Pendragon", that he is referred to in events as "bastard", "royal bastard", or just "Arthur", and after the fact referred to as "Prince" in recognition of his newfound legitimacy (based on merit), given the player hasn't usurped the kingdom from Mordred. Lmk what you think.
3. Noted
4. Noted. I'd written this before I had experimented with the character localisation scopes.
5. Was trying to conjure something up as a reason for why Tewdric would ask Agricola to make the request, and not do it himself. Martial class as in the soldiers who admired Arthur as a great warrior, and would rather he returned to Britain to defend it (per their reaction at the High Council when Agricola made the suggestion - pg. 67, "...The applauding spearmen were warriors of Dumnonia...". If you have alternative suggestions on how to handle this please do say.

Reply:
2. I think that all sounds about right... so probably we just want to go with "bastard son"?
5. Ah okay, I was mostly just thinking that "martial class" is pretty unusual wording. Personally I would just take out that clause, you kind of already have a justification with "Directly announcing your support for the Prince personally will not lend you any favours...".

Reply:
5. Actually, I'll just get rid of that martial class bit. The rest of the text still makes sense without it - it's unneeded.